### PR TITLE
[it] Test --link/--net-alias subcommands

### PR DESCRIPTION
**Pull request by caglar10ur**

### CODE REVIEW

File: tests/test-cases/Group1-Docker-Commands/TestCases.md [hunk 1]
```diff
@@ -47,4 +47,6 @@ Group 1 - Docker Commands
 [Test 1-22 - Docker Volume RM](1-22-Docker-Volume-RM.md)
 -
 [Test 1-23 - Docker Inspect](1-23-Docker-Inspect.md)
--
\ No newline at end of file
+-
+[Test 1-24 - Docker Link](1-24-Docker-Link.md)
```
Comment (emlin) [inline tests/test-cases/Group1-Docker-Commands/TestCases.md]: did you forget 1-24-Docker-Link.md?

Comment (caglar10ur) [inline tests/test-cases/Group1-Docker-Commands/TestCases.md]: I think they generated automatically but I might be wrong, @mhagen-vmware help us :)

Comment (emlin) [inline tests/test-cases/Group1-Docker-Commands/TestCases.md]: I ever manually add md file, but it will better to generate.

Comment (caglar10ur) [inline tests/test-cases/Group1-Docker-Commands/TestCases.md]: OK. let me understand something. Why we are using md files?

**Pull request closed without merging**